cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -174,6 +174,19 @@ spec:
 #        storeType: filestore
 #    - name: "172.17.4.301"
 #      deviceFilter: "^sd."
+  # Ceph supports adding devices as OSDs by Drive Group definitions. TODO: LINK
+  # Rook will give precedence to creating OSDs using Drive Groups before using other methods.
+  driveGroups:
```

Comment:
I still like the idea of doing this in the main `cluster.yaml` and commenting these out. I intend to comment this out, but for my testing they are uncommented. Thoughts?

Reply:
IMO if the example is a big chunk of yaml it makes more sense as a separate yaml, but I'm ok with it staying in cluster.yaml as long as it's not too long. If it's a separate yaml, the docs can also refer to a working example instead of needing to uncomment it. Another thought is to comment out a very short example in cluster.yaml, and have a bigger example in a new yaml.

Reply:
I opted for short example in cluster.yaml and to make a new `cluster-with-drive-groups.yaml`